test/unit/features/transition/transition-group.spec.js

```diff
@@ -293,5 +293,22 @@ if (!isIE9) {
       }).$mount()
       expect('<transition-group> children must be keyed: <div>').toHaveBeenWarned()
     })
+
+    it('component tag in transition group', () => {
+      const component = { template: '<div><slot></slot></div>' }
+      Vue.component('component', component)
```

Comment:
Use a local component here so that it doesn't accidentally pollute other test cases.